src/cli.rs

```diff
@@ -46,6 +46,16 @@ pub enum Commands {
         #[clap(short, long, value_parser = InterfaceParser)]
         name_at_release: InterfaceAtRelease,
     },
+
+    /// Build a JS Slight project
+    Build {
```

Comment:
If this comamnd is JS specific, could we rename it to BuildJS?

Reply:
I figured I'd keep it `build` in case we want to expand to other languages in the future, but I'm good w/ `BuildJs`.